lib/plugins.js

```diff
@@ -31,7 +31,7 @@ const mapValuesWithKey = _.mapValues.convert({
 
 const TRADE_TTL = 2 * T.minutes
 const STALE_TICKER = 3 * T.minutes
-const STALE_BALANCE = 3 * T.minutes
+const STALE_BALANCE = 15 * T.seconds
```

Comment:
We don't really want to change this.

Reply:
Hmm ok, I wasn't 100% sure about this either. Should I try to prevent the coin from staying too long on the screen without messing with the cache time?